odk/odk.py

```diff
@@ -50,20 +50,20 @@ class Product(JsonSchemaMixin):
 
     Here, a product is something that is produced by an ontology workflow.
     A product can be manifested in different formats.
-    
```

Comment:
It would be good to avoid spurious diffs. Do you want to write guidelines and add a linter to travis? How do other projects do this?

Reply:
Ah yeah, sorry about that. I can try and fix my editor from changing whitespace like that. I have no opinion really as to "correct" white space. This is just something my editor does to everyone's annoyance. I don't think we need a linter or anything.